BoostedAnalyzer/src/essentialRecoVarProcessor.cpp

```diff
@@ -4,7 +4,8 @@ using namespace std;
 
 essentialRecoVarProcessor::essentialRecoVarProcessor()
 {
-    pointerToRecoVars.reset(new ReconstructedVars());
+    // reconstruct ttH, reconstruct ttZ
+    pointerToRecoVars.reset(new ReconstructedVars(true, false));
```

Comment:
maybe add Variables, to clarify what `(true, false)`  means

Reply:
Hello sebwieland,
thank you for your comment.

I tried to clarify what these variables mean by adding the comment in the line above. 
But I can also clarify it in the function.